src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_aad_auth.py

```diff
@@ -28,7 +56,7 @@ def __init__(self, *args, **kwargs):
     @ResourceGroupPreparer(parameter_name_for_location='location')
     def test_azconfig_aad_auth(self, resource_group, location):
         aad_store_prefix = get_resource_name_prefix('AADStore')
-        config_store_name = self.create_random_name(prefix=aad_store_prefix, length=36)
+        config_store_name = self.create_random_name(prefix=aad_store_prefix, length=15)
```

Comment:
Why are we updating name lengths for test resources in this PR?

Reply:
We recently split the test into individual files during which we increased the name length to 36 when creating a random name. This however has resulted in some live test that failing as we generated some names that are too long in the case of replicas and keyvault resource so this is being reverted to what they were. In order to avoid delays with merging this fix in, it was included in this PR.  

Prior to this change, the length of the store name for this test was 15.
https://github.com/Azure/azure-cli/blob/cf83ff0089398910a91dc40d1f8b716b4be179de/src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py#L3180

Not sure if there was a reason for this. Would you suggest we change it to 24 for consistency?

Reply:
Thanks, yes please change it to 24 for consistency.

Reply:
This has been updated, thanks!